Bottom: 66af9f25a43a8f9866085890e1093f4ef53378ae
Top:    a5564dd21118c1925c97b3e67df6cec5ac56aa23
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:16 +0000

polygon1.c: Remove now unused memory allocation macro MemGet


---

diff --git a/src/polygon1.c b/src/polygon1.c
index fff6a7a..a5d1325 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -104,10 +104,6 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 
 #define error(code)  longjmp(*(e), code)
 
-#define MemGet(ptr, type) \
-  if (UNLIKELY (((ptr) = (type *)malloc(sizeof(type))) == NULL))	\
-    error(err_no_memory);
-
 #undef DEBUG_LABEL
 #undef DEBUG_ALL_LABELS
 #undef DEBUG_JUMP
